srtcore/buffer.cpp

```diff
@@ -794,7 +794,7 @@ CRcvBuffer::CRcvBuffer(CUnitQueue* queue, int bufsize_pkts)
     , m_iBytesCount(0)
     , m_iAckedPktsCount(0)
     , m_iAckedBytesCount(0)
-    , m_iAvgPayloadSz(7 * 188)
+    , m_uiAvgPayloadSz(7 * 188)
```

Comment:
I think you better use `u` prefix. In the document I tried to summarize and standardize these prefixes, I followed the mostly used `u`. The `ui` prefix is used only for one field: `CUDTSocket::m_uiBacklog`. Not to be confused with a prefix for 64-bit integers, `ull`, with signed version `ll`. There's also one `ul` prefix used in some debug support part in buffers.

Reply:
Renamed to `m_uAvgPayloadSz`